src/promptflow/promptflow/_sdk/_service/apis/telemetry.py

```diff
@@ -0,0 +1,148 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from flask import jsonify, make_response, request
+
+from promptflow._sdk._service import Namespace, Resource
+from promptflow._sdk._service.utils.utils import build_pfs_user_agent, local_user_only
+from promptflow._utils.utils import camel_to_snake
+from promptflow.exceptions import UserErrorException
+
+api = Namespace("Telemetries", description="Telemetry Management")
+
+dict_field = api.schema_model("RunDict", {"additionalProperties": True, "type": "object"})
+
+
+class EventType:
+    START = "Start"
+    END = "End"
+
+
+class AllowedActivityName:
+    FLOW_TEST = "pf.flow.test"
+    FLOW_NODE_TEST = "pf.flow.node_test"
+    GENERATE_TOOL_META = "pf.flow._generate_tools_meta"
+
+
+def _dict_camel_to_snake(data):
+    if isinstance(data, dict):
+        result = {}
+        for key, value in data.items():
+            result[camel_to_snake(key)] = _dict_camel_to_snake(value)
+        return result
+    else:
+        return data
+
+
+def parse_activity_info(telemetry_data, user_agent):
+    from promptflow._sdk._telemetry.activity import generate_request_id
+
+    first_call = telemetry_data.get("firstCall", True)
+    request_id = telemetry_data.get("requestId", generate_request_id())
+
+    return {
+        "request_id": request_id,
```

Comment:
Is there a problem with the naming style? It seems different from other places.
![image](https://github.com/microsoft/promptflow/assets/23182548/3ba3745b-473a-4850-a125-075af4bcc637)

Reply:
It will be converted